tests/warning_list.txt

```diff
@@ -0,0 +1,31 @@
+WARNING: undefined label: 'groupby.transform'
+WARNING: image file not readable: _static/gallery/pandas.png
+WARNING: image file not readable: _static/gallery/numpy.png
+WARNING: image file not readable: _static/gallery/scipy.png
+WARNING: image file not readable: _static/gallery/bokeh.png
+WARNING: image file not readable: _static/gallery/cupy.png
+WARNING: image file not readable: _static/gallery/pyvista.png
+WARNING: image file not readable: _static/gallery/mne-python.png
+WARNING: image file not readable: _static/gallery/networkx.png
+WARNING: image file not readable: _static/gallery/fairlearn.png
+WARNING: image file not readable: _static/gallery/jupyter_book.png
+WARNING: image file not readable: _static/gallery/binder.png
+WARNING: image file not readable: _static/gallery/jupyter.png
+WARNING: image file not readable: _static/gallery/megengine.png
+WARNING: image file not readable: _static/gallery/feature-engine.png
+WARNING: image file not readable: _static/gallery/arviz.png
+WARNING: image file not readable: _static/gallery/sepal.png
+WARNING: image file not readable: _static/gallery/enoslib.png
```

Comment:
I feel like this is going to get tedious because every new image here will create toil where we have to update these warnings. Could we allow this to support glob-like patterns and then do `_static/gallery/*`?

Reply:
the advantage here is that in 1 loop as I pop the element from the list I get both `missing` and `unexpected`. We already have a lot of examples in the gallery do you really expect it to increase that much?

Reply:
maybe not - let's not block this PR on that question, we can revisit if it becomes cumbersome or if people forget.